pkg/diagnose/firewall.go

```diff
@@ -45,6 +45,9 @@ const (
 
 const (
 	clientSourcePort = "9898"
+	loadBalancerName = "submariner-gateway"
+	encapsPortName   = "cable-encaps"
+	nattPortName     = "natt-discovery"
```

Comment:
Similarly, "Auto NAT discovery port"

Reply:
Same here, not sure I understand your comment